driver/ppm_events_public.h

```diff
@@ -998,7 +1043,9 @@ enum ppm_event_type {
 	PPME_SYSCALL_CLONE3_X = 335,
 	PPME_SYSCALL_OPEN_BY_HANDLE_AT_E = 336,
 	PPME_SYSCALL_OPEN_BY_HANDLE_AT_X = 337,
-	PPM_EVENT_MAX = 338
+	PPME_SYSCALL_CAPSET_E = 339,
```

Comment:
338 right? :D

Reply:
Yeah, of course, I am fixing it! By the way, I really don't like having this assignment in a enum. I mean, these constants start from zero and they get increased by one automatically, we don't need to specify the exact value for each one. This may lead to errors like this. I think I am going to propose a PR to fix this if you agree! 😅

Reply:
You know, that bothered me too: https://github.com/falcosecurity/libs/pull/86 :D

Reply:
Cool!! Nice to see that!
BTW just fixed this 😄  thanks a lot